.gitattributes

```diff
@@ -0,0 +1,14 @@
+/.github export-ignore
+/doc export-ignore
+/resources export-ignore
+/tests export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.jane-openapi.php export-ignore
+/.php_cs export-ignore
+/.travis.yml export-ignore
+/CHANGELOG.md export-ignore
+/CONTRIBUTING.md export-ignore
+/Makefile export-ignore
+/phpunit.xml.dist export-ignore
+/VERSIONING.md export-ignore
```

Comment:
Could you sort theses entries?

Thanks

Reply:
They are sorted actually the way I see them in phpstorm. First the folders, then the files. Which order to you want?

Reply:
with a regular sort it will be fine:
```
/.gitattributes export-ignore
/.github export-ignore
/.gitignore export-ignore
/.jane-openapi.php export-ignore
/.php_cs export-ignore
/.travis.yml export-ignore
/CHANGELOG.md export-ignore
/CONTRIBUTING.md export-ignore
/doc export-ignore
/Makefile export-ignore
/phpunit.xml.dist export-ignore
/resources export-ignore
/tests export-ignore
/VERSIONING.md export-ignore
```